MinecraftClient/ChatBot.cs

```diff
@@ -137,6 +137,17 @@ internal void ProcessQueuedText()
         /// <param name="data">The payload for the message</param>
         public virtual void OnPluginMessage(string channel, byte[] data) { }
 
+        public virtual void OnPlayerProperty(Dictionary<string, Double> prop) { }
+
+        public virtual void OnServerTpsUpdate(Double tps) { }
+
+        public virtual void OnEntityMove(Mapping.Entity entity) { }
+
+        public virtual void OnEntitySpawn(Mapping.Entity entity) { }
+
+        public virtual void OnEntityDespawn(int EntityID) { }
```

Comment:
Maybe pass the Entity object here instead of only passing the ID? Maybe the ChatBot will want to do something based on other properties of the Entity being deleted, e.g. watch for entity death without watching for spawn or move.